File: authorization-manager-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/__pluginName__AuthorizationManagerConfig.java
package ${package}.${pluginName};

import se.curity.identityserver.sdk.config.Configuration;
import se.curity.identityserver.sdk.config.annotation.Description;

public interface ${pluginName}AuthorizationManagerConfig extends Configuration
{
    @Description("A parameter used for demo purposes")
    Integer getDemoParameter();
}

File: kotlin-authorization-manager-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__AuthorizationManagerConfig.kt
package ${package}.${pluginName}

import se.curity.identityserver.sdk.config.Configuration
import se.curity.identityserver.sdk.config.annotation.Description

interface ${pluginName}AuthorizationManagerConfig: Configuration
{
    @Description("A parameter used for demo purposes")
    fun getDemoParameter(): Int
}
